<?xml version='1.0' encoding='utf-8'?>
<testsuites name="test_communication.test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug" tests="24" failures="24" errors="0" time="120.227"><testsuite name="test_communication.test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.launch_tests[Arrays]" tests="2" failures="2" errors="0" skipped="0" time="10.01"><testcase classname="test_communication.TestPublisherSubscriber" name="test_subscriber_terminates_in_a_finite_amount_of_time[Arrays]" time="10.008"><failure message="Traceback (most recent call last):&#10;  File &quot;/opt/workspace/build/test_communication/test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.py&quot;, line 66, in test_subscriber_terminates_in_a_finite_amount_of_time&#10;    proc_info.assertWaitForShutdown(process=subscriber_process, timeout=10)&#10;  File &quot;/opt/ros/foxy/lib/python3.8/site-packages/launch_testing/proc_info_handler.py&quot;, line 144, in assertWaitForShutdown&#10;    assert success, &quot;Timed out waiting for process '{}' to finish&quot;.format(process)&#10;AssertionError: Timed out waiting for process '&lt;launch.actions.execute_process.ExecuteProcess object at 0x7fc07eb13730&gt;' to finish&#10;" /></testcase><testcase classname="test_communication.TestPublisherSubscriberAfterShutdown" name="test_processes_finished_gracefully[Arrays]" time="0.002"><failure message="Traceback (most recent call last):&#10;  File &quot;/opt/workspace/build/test_communication/test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.py&quot;, line 74, in test_processes_finished_gracefully&#10;    launch_testing.asserts.assertExitCodes(&#10;  File &quot;/opt/ros/foxy/lib/python3.8/site-packages/launch_testing/asserts/assert_exit_codes.py&quot;, line 62, in assertExitCodes&#10;    assert info.returncode in allowable_exit_codes, 'Proc {} exited with code {}'.format(&#10;AssertionError: Proc test_subscriber-2 exited with code -15&#10;" /></testcase></testsuite><testsuite name="test_communication.test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.launch_tests[BasicTypes]" tests="2" failures="2" errors="0" skipped="0" time="10.008"><testcase classname="test_communication.TestPublisherSubscriber" name="test_subscriber_terminates_in_a_finite_amount_of_time[BasicTypes]" time="10.007"><failure message="Traceback (most recent call last):&#10;  File &quot;/opt/workspace/build/test_communication/test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.py&quot;, line 66, in test_subscriber_terminates_in_a_finite_amount_of_time&#10;    proc_info.assertWaitForShutdown(process=subscriber_process, timeout=10)&#10;  File &quot;/opt/ros/foxy/lib/python3.8/site-packages/launch_testing/proc_info_handler.py&quot;, line 144, in assertWaitForShutdown&#10;    assert success, &quot;Timed out waiting for process '{}' to finish&quot;.format(process)&#10;AssertionError: Timed out waiting for process '&lt;launch.actions.execute_process.ExecuteProcess object at 0x7fc07d292700&gt;' to finish&#10;" /></testcase><testcase classname="test_communication.TestPublisherSubscriberAfterShutdown" name="test_processes_finished_gracefully[BasicTypes]" time="0.002"><failure message="Traceback (most recent call last):&#10;  File &quot;/opt/workspace/build/test_communication/test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.py&quot;, line 74, in test_processes_finished_gracefully&#10;    launch_testing.asserts.assertExitCodes(&#10;  File &quot;/opt/ros/foxy/lib/python3.8/site-packages/launch_testing/asserts/assert_exit_codes.py&quot;, line 62, in assertExitCodes&#10;    assert info.returncode in allowable_exit_codes, 'Proc {} exited with code {}'.format(&#10;AssertionError: Proc test_subscriber-4 exited with code -11&#10;" /></testcase></testsuite><testsuite name="test_communication.test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.launch_tests[BoundedSequences]" tests="2" failures="2" errors="0" skipped="0" time="10.007"><testcase classname="test_communication.TestPublisherSubscriber" name="test_subscriber_terminates_in_a_finite_amount_of_time[BoundedSequences]" time="10.004"><failure message="Traceback (most recent call last):&#10;  File &quot;/opt/workspace/build/test_communication/test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.py&quot;, line 66, in test_subscriber_terminates_in_a_finite_amount_of_time&#10;    proc_info.assertWaitForShutdown(process=subscriber_process, timeout=10)&#10;  File &quot;/opt/ros/foxy/lib/python3.8/site-packages/launch_testing/proc_info_handler.py&quot;, line 144, in assertWaitForShutdown&#10;    assert success, &quot;Timed out waiting for process '{}' to finish&quot;.format(process)&#10;AssertionError: Timed out waiting for process '&lt;launch.actions.execute_process.ExecuteProcess object at 0x7fc07d21aaf0&gt;' to finish&#10;" /></testcase><testcase classname="test_communication.TestPublisherSubscriberAfterShutdown" name="test_processes_finished_gracefully[BoundedSequences]" time="0.002"><failure message="Traceback (most recent call last):&#10;  File &quot;/opt/workspace/build/test_communication/test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.py&quot;, line 74, in test_processes_finished_gracefully&#10;    launch_testing.asserts.assertExitCodes(&#10;  File &quot;/opt/ros/foxy/lib/python3.8/site-packages/launch_testing/asserts/assert_exit_codes.py&quot;, line 62, in assertExitCodes&#10;    assert info.returncode in allowable_exit_codes, 'Proc {} exited with code {}'.format(&#10;AssertionError: Proc test_subscriber-6 exited with code -11&#10;" /></testcase></testsuite><testsuite name="test_communication.test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.launch_tests[Builtins]" tests="2" failures="2" errors="0" skipped="0" time="10.005"><testcase classname="test_communication.TestPublisherSubscriber" name="test_subscriber_terminates_in_a_finite_amount_of_time[Builtins]" time="10.004"><failure message="Traceback (most recent call last):&#10;  File &quot;/opt/workspace/build/test_communication/test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.py&quot;, line 66, in test_subscriber_terminates_in_a_finite_amount_of_time&#10;    proc_info.assertWaitForShutdown(process=subscriber_process, timeout=10)&#10;  File &quot;/opt/ros/foxy/lib/python3.8/site-packages/launch_testing/proc_info_handler.py&quot;, line 144, in assertWaitForShutdown&#10;    assert success, &quot;Timed out waiting for process '{}' to finish&quot;.format(process)&#10;AssertionError: Timed out waiting for process '&lt;launch.actions.execute_process.ExecuteProcess object at 0x7fc07d236be0&gt;' to finish&#10;" /></testcase><testcase classname="test_communication.TestPublisherSubscriberAfterShutdown" name="test_processes_finished_gracefully[Builtins]" time="0.002"><failure message="Traceback (most recent call last):&#10;  File &quot;/opt/workspace/build/test_communication/test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.py&quot;, line 74, in test_processes_finished_gracefully&#10;    launch_testing.asserts.assertExitCodes(&#10;  File &quot;/opt/ros/foxy/lib/python3.8/site-packages/launch_testing/asserts/assert_exit_codes.py&quot;, line 62, in assertExitCodes&#10;    assert info.returncode in allowable_exit_codes, 'Proc {} exited with code {}'.format(&#10;AssertionError: Proc test_subscriber-8 exited with code -11&#10;" /></testcase></testsuite><testsuite name="test_communication.test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.launch_tests[Constants]" tests="2" failures="2" errors="0" skipped="0" time="10.14"><testcase classname="test_communication.TestPublisherSubscriber" name="test_subscriber_terminates_in_a_finite_amount_of_time[Constants]" time="10.005"><failure message="Traceback (most recent call last):&#10;  File &quot;/opt/workspace/build/test_communication/test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.py&quot;, line 66, in test_subscriber_terminates_in_a_finite_amount_of_time&#10;    proc_info.assertWaitForShutdown(process=subscriber_process, timeout=10)&#10;  File &quot;/opt/ros/foxy/lib/python3.8/site-packages/launch_testing/proc_info_handler.py&quot;, line 144, in assertWaitForShutdown&#10;    assert success, &quot;Timed out waiting for process '{}' to finish&quot;.format(process)&#10;AssertionError: Timed out waiting for process '&lt;launch.actions.execute_process.ExecuteProcess object at 0x7fc07d252400&gt;' to finish&#10;" /></testcase><testcase classname="test_communication.TestPublisherSubscriberAfterShutdown" name="test_processes_finished_gracefully[Constants]" time="0.135"><failure message="Traceback (most recent call last):&#10;  File &quot;/opt/workspace/build/test_communication/test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.py&quot;, line 74, in test_processes_finished_gracefully&#10;    launch_testing.asserts.assertExitCodes(&#10;  File &quot;/opt/ros/foxy/lib/python3.8/site-packages/launch_testing/asserts/assert_exit_codes.py&quot;, line 62, in assertExitCodes&#10;    assert info.returncode in allowable_exit_codes, 'Proc {} exited with code {}'.format(&#10;AssertionError: Proc test_subscriber-10 exited with code -11&#10;" /></testcase></testsuite><testsuite name="test_communication.test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.launch_tests[Defaults]" tests="2" failures="2" errors="0" skipped="0" time="10.007"><testcase classname="test_communication.TestPublisherSubscriber" name="test_subscriber_terminates_in_a_finite_amount_of_time[Defaults]" time="10.004"><failure message="Traceback (most recent call last):&#10;  File &quot;/opt/workspace/build/test_communication/test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.py&quot;, line 66, in test_subscriber_terminates_in_a_finite_amount_of_time&#10;    proc_info.assertWaitForShutdown(process=subscriber_process, timeout=10)&#10;  File &quot;/opt/ros/foxy/lib/python3.8/site-packages/launch_testing/proc_info_handler.py&quot;, line 144, in assertWaitForShutdown&#10;    assert success, &quot;Timed out waiting for process '{}' to finish&quot;.format(process)&#10;AssertionError: Timed out waiting for process '&lt;launch.actions.execute_process.ExecuteProcess object at 0x7fc07d1ead00&gt;' to finish&#10;" /></testcase><testcase classname="test_communication.TestPublisherSubscriberAfterShutdown" name="test_processes_finished_gracefully[Defaults]" time="0.002"><failure message="Traceback (most recent call last):&#10;  File &quot;/opt/workspace/build/test_communication/test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.py&quot;, line 74, in test_processes_finished_gracefully&#10;    launch_testing.asserts.assertExitCodes(&#10;  File &quot;/opt/ros/foxy/lib/python3.8/site-packages/launch_testing/asserts/assert_exit_codes.py&quot;, line 62, in assertExitCodes&#10;    assert info.returncode in allowable_exit_codes, 'Proc {} exited with code {}'.format(&#10;AssertionError: Proc test_subscriber-12 exited with code -11&#10;" /></testcase></testsuite><testsuite name="test_communication.test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.launch_tests[Empty]" tests="2" failures="2" errors="0" skipped="0" time="10.01"><testcase classname="test_communication.TestPublisherSubscriber" name="test_subscriber_terminates_in_a_finite_amount_of_time[Empty]" time="10.008"><failure message="Traceback (most recent call last):&#10;  File &quot;/opt/workspace/build/test_communication/test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.py&quot;, line 66, in test_subscriber_terminates_in_a_finite_amount_of_time&#10;    proc_info.assertWaitForShutdown(process=subscriber_process, timeout=10)&#10;  File &quot;/opt/ros/foxy/lib/python3.8/site-packages/launch_testing/proc_info_handler.py&quot;, line 144, in assertWaitForShutdown&#10;    assert success, &quot;Timed out waiting for process '{}' to finish&quot;.format(process)&#10;AssertionError: Timed out waiting for process '&lt;launch.actions.execute_process.ExecuteProcess object at 0x7fc07d2065e0&gt;' to finish&#10;" /></testcase><testcase classname="test_communication.TestPublisherSubscriberAfterShutdown" name="test_processes_finished_gracefully[Empty]" time="0.002"><failure message="Traceback (most recent call last):&#10;  File &quot;/opt/workspace/build/test_communication/test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.py&quot;, line 74, in test_processes_finished_gracefully&#10;    launch_testing.asserts.assertExitCodes(&#10;  File &quot;/opt/ros/foxy/lib/python3.8/site-packages/launch_testing/asserts/assert_exit_codes.py&quot;, line 62, in assertExitCodes&#10;    assert info.returncode in allowable_exit_codes, 'Proc {} exited with code {}'.format(&#10;AssertionError: Proc test_subscriber-14 exited with code -11&#10;" /></testcase></testsuite><testsuite name="test_communication.test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.launch_tests[MultiNested]" tests="2" failures="2" errors="0" skipped="0" time="10.009"><testcase classname="test_communication.TestPublisherSubscriber" name="test_subscriber_terminates_in_a_finite_amount_of_time[MultiNested]" time="10.007"><failure message="Traceback (most recent call last):&#10;  File &quot;/opt/workspace/build/test_communication/test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.py&quot;, line 66, in test_subscriber_terminates_in_a_finite_amount_of_time&#10;    proc_info.assertWaitForShutdown(process=subscriber_process, timeout=10)&#10;  File &quot;/opt/ros/foxy/lib/python3.8/site-packages/launch_testing/proc_info_handler.py&quot;, line 144, in assertWaitForShutdown&#10;    assert success, &quot;Timed out waiting for process '{}' to finish&quot;.format(process)&#10;AssertionError: Timed out waiting for process '&lt;launch.actions.execute_process.ExecuteProcess object at 0x7fc07d19be80&gt;' to finish&#10;" /></testcase><testcase classname="test_communication.TestPublisherSubscriberAfterShutdown" name="test_processes_finished_gracefully[MultiNested]" time="0.002"><failure message="Traceback (most recent call last):&#10;  File &quot;/opt/workspace/build/test_communication/test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.py&quot;, line 74, in test_processes_finished_gracefully&#10;    launch_testing.asserts.assertExitCodes(&#10;  File &quot;/opt/ros/foxy/lib/python3.8/site-packages/launch_testing/asserts/assert_exit_codes.py&quot;, line 62, in assertExitCodes&#10;    assert info.returncode in allowable_exit_codes, 'Proc {} exited with code {}'.format(&#10;AssertionError: Proc test_subscriber-16 exited with code -11&#10;" /></testcase></testsuite><testsuite name="test_communication.test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.launch_tests[Nested]" tests="2" failures="2" errors="0" skipped="0" time="10.008"><testcase classname="test_communication.TestPublisherSubscriber" name="test_subscriber_terminates_in_a_finite_amount_of_time[Nested]" time="10.006"><failure message="Traceback (most recent call last):&#10;  File &quot;/opt/workspace/build/test_communication/test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.py&quot;, line 66, in test_subscriber_terminates_in_a_finite_amount_of_time&#10;    proc_info.assertWaitForShutdown(process=subscriber_process, timeout=10)&#10;  File &quot;/opt/ros/foxy/lib/python3.8/site-packages/launch_testing/proc_info_handler.py&quot;, line 144, in assertWaitForShutdown&#10;    assert success, &quot;Timed out waiting for process '{}' to finish&quot;.format(process)&#10;AssertionError: Timed out waiting for process '&lt;launch.actions.execute_process.ExecuteProcess object at 0x7fc07d1ba760&gt;' to finish&#10;" /></testcase><testcase classname="test_communication.TestPublisherSubscriberAfterShutdown" name="test_processes_finished_gracefully[Nested]" time="0.002"><failure message="Traceback (most recent call last):&#10;  File &quot;/opt/workspace/build/test_communication/test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.py&quot;, line 74, in test_processes_finished_gracefully&#10;    launch_testing.asserts.assertExitCodes(&#10;  File &quot;/opt/ros/foxy/lib/python3.8/site-packages/launch_testing/asserts/assert_exit_codes.py&quot;, line 62, in assertExitCodes&#10;    assert info.returncode in allowable_exit_codes, 'Proc {} exited with code {}'.format(&#10;AssertionError: Proc test_subscriber-18 exited with code -11&#10;" /></testcase></testsuite><testsuite name="test_communication.test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.launch_tests[Strings]" tests="2" failures="2" errors="0" skipped="0" time="10.007"><testcase classname="test_communication.TestPublisherSubscriber" name="test_subscriber_terminates_in_a_finite_amount_of_time[Strings]" time="10.006"><failure message="Traceback (most recent call last):&#10;  File &quot;/opt/workspace/build/test_communication/test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.py&quot;, line 66, in test_subscriber_terminates_in_a_finite_amount_of_time&#10;    proc_info.assertWaitForShutdown(process=subscriber_process, timeout=10)&#10;  File &quot;/opt/ros/foxy/lib/python3.8/site-packages/launch_testing/proc_info_handler.py&quot;, line 144, in assertWaitForShutdown&#10;    assert success, &quot;Timed out waiting for process '{}' to finish&quot;.format(process)&#10;AssertionError: Timed out waiting for process '&lt;launch.actions.execute_process.ExecuteProcess object at 0x7fc07d15eac0&gt;' to finish&#10;" /></testcase><testcase classname="test_communication.TestPublisherSubscriberAfterShutdown" name="test_processes_finished_gracefully[Strings]" time="0.002"><failure message="Traceback (most recent call last):&#10;  File &quot;/opt/workspace/build/test_communication/test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.py&quot;, line 74, in test_processes_finished_gracefully&#10;    launch_testing.asserts.assertExitCodes(&#10;  File &quot;/opt/ros/foxy/lib/python3.8/site-packages/launch_testing/asserts/assert_exit_codes.py&quot;, line 62, in assertExitCodes&#10;    assert info.returncode in allowable_exit_codes, 'Proc {} exited with code {}'.format(&#10;AssertionError: Proc test_subscriber-20 exited with code -11&#10;" /></testcase></testsuite><testsuite name="test_communication.test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.launch_tests[UnboundedSequences]" tests="2" failures="2" errors="0" skipped="0" time="10.007"><testcase classname="test_communication.TestPublisherSubscriber" name="test_subscriber_terminates_in_a_finite_amount_of_time[UnboundedSequences]" time="10.006"><failure message="Traceback (most recent call last):&#10;  File &quot;/opt/workspace/build/test_communication/test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.py&quot;, line 66, in test_subscriber_terminates_in_a_finite_amount_of_time&#10;    proc_info.assertWaitForShutdown(process=subscriber_process, timeout=10)&#10;  File &quot;/opt/ros/foxy/lib/python3.8/site-packages/launch_testing/proc_info_handler.py&quot;, line 144, in assertWaitForShutdown&#10;    assert success, &quot;Timed out waiting for process '{}' to finish&quot;.format(process)&#10;AssertionError: Timed out waiting for process '&lt;launch.actions.execute_process.ExecuteProcess object at 0x7fc07d182040&gt;' to finish&#10;" /></testcase><testcase classname="test_communication.TestPublisherSubscriberAfterShutdown" name="test_processes_finished_gracefully[UnboundedSequences]" time="0.002"><failure message="Traceback (most recent call last):&#10;  File &quot;/opt/workspace/build/test_communication/test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.py&quot;, line 74, in test_processes_finished_gracefully&#10;    launch_testing.asserts.assertExitCodes(&#10;  File &quot;/opt/ros/foxy/lib/python3.8/site-packages/launch_testing/asserts/assert_exit_codes.py&quot;, line 62, in assertExitCodes&#10;    assert info.returncode in allowable_exit_codes, 'Proc {} exited with code {}'.format(&#10;AssertionError: Proc test_subscriber-22 exited with code -11&#10;" /></testcase></testsuite><testsuite name="test_communication.test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.launch_tests[WStrings]" tests="2" failures="2" errors="0" skipped="0" time="10.007"><testcase classname="test_communication.TestPublisherSubscriber" name="test_subscriber_terminates_in_a_finite_amount_of_time[WStrings]" time="10.006"><failure message="Traceback (most recent call last):&#10;  File &quot;/opt/workspace/build/test_communication/test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.py&quot;, line 66, in test_subscriber_terminates_in_a_finite_amount_of_time&#10;    proc_info.assertWaitForShutdown(process=subscriber_process, timeout=10)&#10;  File &quot;/opt/ros/foxy/lib/python3.8/site-packages/launch_testing/proc_info_handler.py&quot;, line 144, in assertWaitForShutdown&#10;    assert success, &quot;Timed out waiting for process '{}' to finish&quot;.format(process)&#10;AssertionError: Timed out waiting for process '&lt;launch.actions.execute_process.ExecuteProcess object at 0x7fc07d124790&gt;' to finish&#10;" /></testcase><testcase classname="test_communication.TestPublisherSubscriberAfterShutdown" name="test_processes_finished_gracefully[WStrings]" time="0.002"><failure message="Traceback (most recent call last):&#10;  File &quot;/opt/workspace/build/test_communication/test_publisher_subscriber__rclpy__rclcpp__rmw_fastrtps_cpp__rmw_opendds_cpp_Debug.py&quot;, line 74, in test_processes_finished_gracefully&#10;    launch_testing.asserts.assertExitCodes(&#10;  File &quot;/opt/ros/foxy/lib/python3.8/site-packages/launch_testing/asserts/assert_exit_codes.py&quot;, line 62, in assertExitCodes&#10;    assert info.returncode in allowable_exit_codes, 'Proc {} exited with code {}'.format(&#10;AssertionError: Proc test_subscriber-24 exited with code -11&#10;" /></testcase></testsuite></testsuites>